tests/test_things_example.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import Mock
+
+from examples.things import app, things, ThingsResource
+import falcon
+
+
+def test_app():
+    assert app
+    resource, *_ = app._router.find('/things')
+    assert resource is things
+
+
+def test_things():
+    assert isinstance(things, ThingsResource)
+
+
+def test_ThingsResource_on_get():
```

Comment:
Hi @Ciemaar, thanks for contributing! Tests like these are often neglected and really important for our users.

You might find this a simpler way to write tests for resources:


```python
import falcon.testing as testing


# (Optional) First create a pytest fixture which returns a TestClient which 
# can be used to send requests to the passed application
@pytest.fixture()
def client():
    return testing.TestClient(app)


# add the fixture name, in this case "client" as an argument to the test. 
# pytest will see that and inject the fixture you created above.
#
# NOTE: you can create the `client` in the test if you want but a 
# fixture lends itself to better code reuse.
def test_things_resource_response(client):
    # client = testing.TestClient(app)
    # Then use the client to simulate a request to the api.
    resp = client.simulate_get('/things')

    # Then you can assert as you did before.
    assert resp.status_code == 200
    assert resp.text == (
        '\nTwo things awe me most, the starry sky above me and the moral law within me.'
        '\n\n    ~ Immanuel Kant\n\n'
    )
```